contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -621,18 +611,27 @@ contract LidoOracle is ILidoOracle, AragonApp {
         internal
         view
     {
-        if (_postTotalPooledEther >= _preTotalPooledEther) {  // check profit constraint
-            uint256 reward = _postTotalPooledEther - _preTotalPooledEther;
-            uint256 allowedBeaconBalanceAnnualIncrease =
-                ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE_POSITION.getStorageUint256().mul(_preTotalPooledEther);
-            uint256 rewardAnnualized = uint256(10000 * 365 days).mul(reward).div(_timeElapsed);
-            require(rewardAnnualized <= allowedBeaconBalanceAnnualIncrease, "ALLOWED_BEACON_BALANCE_INCREASE");
-        } else {  // check loss constraint
-            uint256 delta = _preTotalPooledEther - _postTotalPooledEther;
-            uint256 allowedBeaconBalanceDecrease =
-                ALLOWED_BEACON_BALANCE_RELATIVE_DECREASE_POSITION.getStorageUint256().mul(_preTotalPooledEther);
-            uint256 loss = uint256(10000).mul(delta);
-            require(loss <= allowedBeaconBalanceDecrease, "ALLOWED_BEACON_BALANCE_DECREASE");
+        if (_postTotalPooledEther >= _preTotalPooledEther) {
+            // increase                        = _postTotalPooledEther - _preTotalPooledEther,
+            // relativeIncrease                = increase / _preTotalPooledEther,
+            // annualRelativeIncrease          = relativeIncrease / (timeElapsed / 365 days),
+            // annualRelativeIncreasePermyriad = annualRelativeIncrease * 10000
+            uint256 allowedAnnualRelativeIncreasPermyriad =
```

Comment:
We are using BP (basis points) in another contracts (e.g Lido.sol, StETH.sol)
Permyriad is very rare word and needs to be googled any time

Reply:
Replaced Permyriad -> Bp (Basis points), thanks